tests/foreman/cli/test_puppetmodule.py

```diff
@@ -24,30 +24,33 @@
 from robottelo.config import settings
 from robottelo.constants.repos import FAKE_0_PUPPET_REPO
 from robottelo.constants.repos import FAKE_1_PUPPET_REPO
-from robottelo.test import CLITestCase
 
 
-class PuppetModuleTestCase(CLITestCase):
-    """Tests for PuppetModule via Hammer CLI"""
+@pytest.fixture(scope='module')
+def make_setup():
+    org = make_org()
```

Comment:
Consider using `module_org` fixture, which is available without import just by using the name:

https://github.com/SatelliteQE/robottelo/blob/master/pytest_fixtures/api_fixtures.py#L45

Reply:
Reconsidered and used, thank you.

Reply:
Umm.. one more thing - in the end I didn't pass the `module_org` fixture directly to the tests but through the `module_setup`, because I need the setup to be called even if the test uses only the `module_org`. Yes, I could pass both fixtures, but then the `module_setup` would look like unused in the test.

Reply:
That's a fine pattern to follow. When using pytest, if you want to see the full 'stack' of fixtures in use by a test, you should use `pytest --setup-plan tests/path/test_module.py`, which will end up showing all fixtures from the framework that are in use.  Its not necessary, or a good pattern, to try and include every fixture as a test function parameter, when the fixture return isn't directly used.